### PR TITLE
Asset Header

### PR DESCRIPTION
**Pull request by rleshner**
Styled the asset header to look a bit more profesh

### CODE REVIEW

File: src/elm/SelectedAsset.elm [hunk 1]
```diff
@@ -203,12 +203,25 @@ view : ContractSet -> Account -> TokenState -> CompoundState -> OracleState -> P
...
+                    div [ class "asset" ] [ 
+                        div [ class "name" ] [ text token.name ]
+                        , div [ class "symbol" ] [ 
+                            text token.symbol 
+                            , augmentTokenButton token account contract ]
                     ]
+
+                , div [ class "faucet-approve-div" ] [ tokenApprovalStateButton customerAddress moneyMarket token (not (tokenIsApproved token tokenState)) ]
+                ]
+
```
Comment (hayesgm) [inline src/elm/SelectedAsset.elm]: Why all da space?

Comment (rleshner) [reply src/elm/SelectedAsset.elm]: It's how I like to whitespace Elm. Feel free to change if you have another style preference!

Comment (rleshner) [reply src/elm/SelectedAsset.elm]: It's like HTML / markup whitespacing! So much less confusing

**Comment by hayesgm** [comment]
much better.

**Pull request merged**